### PR TITLE
no-waiting option

### PR DESCRIPTION
**Pull request by ablasdel**
PEP8 & de-duplication separated into their own commits to make review easier.

### CODE REVIEW

File: src/moveit_python/pick_place_interface.py [hunk 1]
```diff
@@ -57,22 +63,34 @@ def __init__(self, group = "arm", ee_group = "gripper", plan_only = False, verbo
...
+        return self._pick_action
+
+    def get_place_action(self):
+        return self._place_action
+
+
+    def verify_arguments(self, arguments):
         supported_args = ("allow_gripper_support_collision",
                           "allowed_touch_objects",
+                          "goal_is_eef",
```
Comment (mikeferguson) [inline src/moveit_python/pick_place_interface.py]: this is not ideal, goal_is_eef should not be valid for pick

**Comment by mikeferguson** [comment]
Most of this looks good -- however I'm not convinced that combining the pick/place together in general_retry is such a good idea -- the code logic is quite a bit longer/complex since there are so many if/else statements.

**Comment by ablasdel** [comment]
That makes sense actually. my pep8 linter complains that there are too many branches in that bit of code.

**Comment by mikeferguson** [comment]
Yeah, please revise that piece of code and make a comment here when it is done so -- you probably will need to do a bit of gitfu on the last commit or two to keep some clean history

**Comment by ablasdel** [comment]
Honestly once it is reviewed I think squashing this all into one commit is the better option

**Comment by mikeferguson** [comment]
I would suggest leaving the initial PEP8 updates as a separate commit -- makes it easier to see actual changes versus formatting.

**Comment by ablasdel** [comment]
hacked out the code de-duplication instead of splitting it up. I can do another pass later if needed. Note the 2 "meat" commits are now squashed into 52e3900

**Pull request merged**